x-pack/plugins/infra/public/pages/metrics/hosts/components/tabs/metrics/metrics_grid.tsx

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React from 'react';
+
+import { EuiFlexGrid, EuiFlexItem, EuiFlexGroup, EuiText, EuiI18n } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { MetricChart } from './metric_chart';
+
+const DEFAULT_BREAKDOWN_SIZE = 20;
+
+export const MetricsGrid = () => {
+  return (
+    <>
+      <EuiFlexGroup direction="column" gutterSize="s" data-test-subj="hostsView-metricChart">
+        <EuiFlexItem>
+          <EuiFlexGroup justifyContent="spaceBetween" wrap={false} alignItems="center">
+            <EuiFlexItem grow={false} style={{ flex: 1 }}>
+              <EuiText size="xs">
+                <EuiI18n
+                  token="xpack.infra.hostsViewPage.tabs.metricsCharts.cpuCores"
+                  default="Showing for Top {maxHosts} hosts by {attribute}"
+                  values={{
+                    maxHosts: <strong>{DEFAULT_BREAKDOWN_SIZE}</strong>,
+                    attribute: <strong>name</strong>,
+                  }}
+                />
+              </EuiText>
+            </EuiFlexItem>
+          </EuiFlexGroup>
+        </EuiFlexItem>
+        <EuiFlexItem>
+          <EuiFlexGrid columns={2} gutterSize="s">
+            <EuiFlexItem>
+              <MetricChart
+                type="load"
+                title={i18n.translate('xpack.infra.hostsViewPage.tabs.metricsCharts.load', {
+                  defaultMessage: 'Normalized Load',
+                })}
+                breakdownSize={DEFAULT_BREAKDOWN_SIZE}
+              />
+            </EuiFlexItem>
```

Comment:
**suggestion:** not a logic change, but I found it a bit longer to find a specific metric reading this return statement.
 You could maybe extract this settings in a list out of this component scope (so you also don't compute the translation with `i18n.translate` on each render) and you can then loop on it to render each chart, something like this:
```ts
const charts = [
  {
    type: 'load',
    title: i18n.translate('xpack.infra.hostsViewPage.tabs.metricsCharts.load', {
      defaultMessage: 'Normalized Load',
    }),
  },
  {
    type: 'cpu',
    title: i18n.translate('xpack.infra.hostsViewPage.tabs.metricsCharts.cpu', {
      defaultMessage: 'CPU Usage',
    }),
  },
  // others
];

// Then inside the component
<EuiFlexGrid columns={2} gutterSize="s">
  {charts.map((chart) => (
    <EuiFlexItem>
      <MetricChart breakdownSize={DEFAULT_BREAKDOWN_SIZE} {...chart} />
    </EuiFlexItem>
  ))}
</EuiFlexGrid>
```

Reply:
Yeah, I see where you're coming from. This makes sense.